pkgs/development/python-modules/mkdocs-rss-plugin/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage {
+  pname = "mkdocs-rss-plugin";
+  version = "1.8.0";
```

```suggestion
  version = "1.8.0";
  pyproject = true;

  disabled = pythonOlder "3.8";
```

see https://github.com/Guts/mkdocs-rss-plugin/blob/main/setup.py#L82